pkg/build/builder/cmd/builder.go

```diff
@@ -39,6 +39,7 @@ func run(builderFactory factoryFunc, scmAuths []scmauth.SCMAuth) {
 		glog.Fatalf("Error obtaining docker client: %v", err)
 	}
 	buildStr := os.Getenv("BUILD")
+	glog.Infof("$BUILD env var is %s \n", buildStr)
```

Comment:
There was nothing dumping this from my repeated attempts.  So I'd like to keep this.  Will make it a V(4).